readthedocs/restapi/views/integrations.py

```diff
@@ -173,6 +202,7 @@ class GitHubWebhookView(WebhookMixin, APIView):
     """
 
     integration_type = Integration.GITHUB_WEBHOOK
+    invalid_payload_msg = 'Payload not valid, invalid or missing secret'
```

Comment:
I'd re-write it like this: "Payload's signature is not valid or missing."

Reply:
We don't have a signature here, just a secret (like the token validation for generic webhooks). So, that's part of the request, not for the payload.

Reply:
We are computing the signature with hmac and the whole boyd at `is_payload_valid`. That's what I'm trying to communicate to the user.

Reply:
That's for github only, this message is for gitlab

Reply:
ok, I got confused this if for GitHub

Reply:
:)